### PR TITLE
[feature-betareview] global: fixed the files diff script

### PR DESCRIPTION
**Pull request by krzysztof**
Signed-off-by: Krzysztof Nowak <k.nowak@cern.ch>

### CODE REVIEW

**Comment by krzysztof** [comment]
Those files will differ by date in the license, it's more convenient to see the diff.

**Comment by lnielsen** [comment]
Even if you set the correct date when creating the cookiecutter output?

**Comment by krzysztof** [comment]
@lnielsen Some files will have both '2015, 2016'

**Comment by lnielsen** [comment]
It's a lot easier if you can just copy them over, so can't we just rely on ``git diff`` on the repo instead?

**Comment by krzysztof** [comment]
@lnielsen Added copyright year skipping instead

File: scripts/repocheck.py [hunk 1]
```diff
@@ -97,8 +97,7 @@
   'contact',
   'contact-email',
   'url',
-  'license',
-  'description',
```
Comment (krzysztof) [inline scripts/repocheck.py]: @lnielsen Should description be checked for here? The only way to pass this test is to copy-paste the description from the setup.py into the CC boostrap (at which point this tests is not catching anything meaningful?).

Comment (lnielsen) [inline scripts/repocheck.py]: Cookiecutter pastes the project description into multiple files. This ensures that the module description is correctly set in the setup.py.

**Pull request merged**